Neos.FluidAdaptor/Classes/ViewHelpers/Format/BytesViewHelper.php

```diff
@@ -28,18 +30,61 @@
  * // depending on the value of {fileSize}
  * </output>
  *
- * <code title="Defaults">
+ * <code title="With all parameters">
  * {fileSize -> f:format.bytes(decimals: 2, decimalSeparator: ',', thousandsSeparator: ',')}
  * </code>
  * <output>
  * 1,023.00 B
  * // depending on the value of {fileSize}
  * </output>
  *
+ * <code title="Inline notation with current locale used">
+ * {fileSize -> f:format.bytes(forceLocale: true)}
+ * </code>
+ * <output>
+ * 6.543,21 KB
+ * // depending on the value of {fileSize} and the current locale
+ * </output>
+ *
+ * <code title="Inline notation with specific locale used">
+ * {fileSize -> f:format.bytes(forceLocale: 'de_CH')}
+ * </code>
+ * <output>
+ * 1'337.42 MB
+ * // depending on the value of {fileSize}
+ * </output>
+ *
  * @api
  */
-class BytesViewHelper extends AbstractViewHelper
+class BytesViewHelper extends AbstractLocaleAwareViewHelper
 {
+    /**
+     * Supported file size units
+     */
+    protected const SIZE_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+    /**
+     * @Flow\Inject
```

Comment:
Because you inject a property, you must add

use Neos\Flow\Annotations as Flow

To your top "use" statements. Otherwise, Flow throws the exception you see in your test results.